dev/NavigationView/NavigationView.cpp

```diff
@@ -367,6 +368,24 @@ void NavigationView::OnApplyTemplate()
 
     m_accessKeyInvokedRevoker = AccessKeyInvoked(winrt::auto_revoke, { this, &NavigationView::OnAccessKeyInvoked });
 
+    if (SharedHelpers::IsThemeShadowAvailable())
+    {
+#if defined(BUILD_WINDOWS)
+        if (auto splitView = m_rootSplitView.get())
+        {
+            if (auto contentRoot = splitView.Content())
```

Comment:
>Content [](start = 45, length = 7)

Please confirm with Yulia, will the customer change the content directly? If yes, we may lose shadow after customer navigate to another page

Reply:
I don't think splitView is public @YuliKl ?

Reply:
yes. splitview.content is not directly binding with nav.Content, and it should be not a problem.

---
In reply to: [242389086](https://github.com/Microsoft/microsoft-ui-xaml/pull/96#discussion_r242389086) [](ancestors = 242389086)